tool_collections/taxonomy/find_diag_hits/tool_dependencies.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0"?>
-<tool_dependency>
-  <package name="taxonomy" version="1.0.0">
-      <repository name="package_taxonomy_1_0_0" owner="devteam" prior_installation_required="False" />
```

Comment:
Is there a conda package for this `taxonomy`?

Reply:
Oh dear, I don't think there is, that one will have to come back.

Reply:
Turns out the toolshed's `package_taxonomy_1_0_0` is trying to download from a URI that no longer exists. I'm currently investigating whether @spond's `gb_taxonomy_tools` version 1.0.1 is an appropriate replacement.